crates/iceberg/tests/file_io_s3_test.rs

```diff
@@ -49,12 +49,12 @@ async fn get_file_io() -> FileIO {
 
     let guard = DOCKER_COMPOSE_ENV.read().unwrap();
     let docker_compose = guard.as_ref().unwrap();
-    let container_ip = docker_compose.get_container_ip("minio");
```

Comment:
I feel like we can use the docker's bridge ip instead, but I'm not sure since I don't have a mac.